plugins/rapid7_insightidr/komand_rapid7_insightidr/util/constants.py

```diff
@@ -0,0 +1,8 @@
+# Constants for query/action.py
+
+# The way data indexing works changed on the 24/11/2022.
+# For any search with most_recent_first=true 'from' must not be older than 24/11/2022
+twenty_fourth_november = 1669248000
```

Comment:
In general constants are written capitalized
```suggestion
TWENTY_FOURTH_NOVEMBER = 1669248000
```